Documentation/network/servicemesh/gateway-api/gateway-api.rst

```diff
@@ -33,6 +33,11 @@ tests are passed.
 - `TLSRoute (experimental) <https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io/v1alpha2.TLSRoute>`__
 - `ReferenceGrant <https://gateway-api.sigs.k8s.io/api-types/referencegrant/>`_
 
+.. admonition:: Video
+ :class: attention
+
+  If you'd like more insights on Cilium's Gateway API support, check out `Nick Young's explanation in eCHO episode 58 <https://www.youtube.com/watch?v=60epwCxO8G4&index=80&t=2024s>`__.
```

Comment:
I'd maybe keep the name of the presenter out of this, for fairness and consistency? This seems to be the only occurrence where you introduce the speaker?

Reply:
fair enough, will fix